src/pilot/config_idrac.py

```diff
@@ -211,13 +211,13 @@ def config_boot_mode(drac_client, ip_service_tag, node, boot_mode):
     response = drac_client.set_bios_settings(settings)
 
     job_id = None
-    if response['commit_required']:
+    if response['is_commit_required']:
         job_id = drac_client.commit_pending_bios_changes(reboot=False,
                                                          start_time=None)
 
     # Note that "commit_required" is actually "reboot_required" under the
```

Comment:
This comment is confusing now, should it be removed?

Reply:
yes.  since commit_required has been removed, we should remove this comment too.